src/Bicep.LangServer/Snippets/SnippetsProvider.cs

```diff
@@ -251,14 +253,11 @@ public IEnumerable<Snippet> GetResourceBodyCompletionSnippets(TypeSymbol typeSym
 
         private IEnumerable<Snippet> GetResourceBodyCompletionSnippetFromAzTypes(TypeSymbol typeSymbol)
         {
-            string description = "Required properties";
-
             if (typeSymbol is ResourceType resourceType)
             {
                 if (resourceType.Body is ObjectType objectType)
                 {
-                    string label = "required-properties";
-                    Snippet? snippet = GetRequiredPropertiesSnippet(objectType, label, description);
+                    Snippet? snippet = GetRequiredPropertiesSnippet(objectType, RequiredPropertiesLabel);
```

Comment:
The logic here looks similar to `GetObjectBodyCompletionSnippet()` - can we call into that method instead of duplicating?

Reply:
Yes, I cleaned it up here:
https://github.com/Azure/bicep/commit/750d9e4726e90b21f42c97565f6d224fc85c0963

Thanks!